scripts/scripts.js

```diff
@@ -1,617 +1,20 @@
-/*
- * Copyright 2021 Adobe. All rights reserved.
- * This file is licensed to you under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License. You may obtain a copy
- * of the License at http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software distributed under
- * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
- * OF ANY KIND, either express or implied. See the License for the specific language
- * governing permissions and limitations under the License.
- */
```

Comment:
why did you remove the copyright?

Reply:
Probably to indicate that this is project code, not product code.

Reply:
yeah, i am not entirely sure why i left it out, but i like @rofe's argument ;)